docs/release_notes/release_notes.md

```diff
@@ -8,61 +8,73 @@ has_children: true
 
 # Release Notes
 
-## Version 2.2.0 (June 2022)
+## Version 2.3.0 (July 2022)
 
+* ### New Cyences_Vulnerabilities and Cyences_Assets datamodel
 
-* ### New Windows Dashboard 
-    * Added `Windows Patch` dashboard to monitor for Windows update related events.
+    * Added `Cyences_Vulnerabilities` and `Cyences_Assets` datamodel.
 
-* ### Added New Alerts
+    * Added field extraction, eventtypes and tags for Qualys, Tenable IO, Tenable Nessus and CrowdStrike Spotlight vulnerability events to map with `Cyences_Vulnerabilities` datamodel.
 
-   * Active Directory
-      * `AD - Multiple Password Changes in Short Time Period` 
-      * `AD - Bulk User Creation or Deletion` 
-           
-   * G Suite
-      * `G Suite - Multiple Password Changes in Short Time Period` 
-      * `G Suite - Bulk User Creation or Deletion` 
+    * Added field extraction, eventtypes and tags for Qualys and Tenable asset events to map with `Cyences_Assets` datamodel.
 
-* ### Enhancements 
+    * Added `cs_all_vuln` and `cs_all_assets` KV lookup.
 
-    * Active Directory 
-        * Made improvements to the `AD - Password Change Outside Working Hour` alert and dashboard panel to display additional fields.
+    * Added `Asset Inventory - Vulnerability Lookup Gen` and `Asset Inventory - Lookup Gen` to populate `cs_all_assets` and `cs_all_vuln` lookups from `Cyences_Vulnerabilities` and `Cyences_Assets` datamodel respectively.
 
-    * G Suite
-        * Added `User Created` and `User Deleted` dashboard panels to the G Suite dashboard.
+    * Updated `Lansweeper` and `Network Reports` dashboard to use `cs_all_vuln` and `cs_all_assets` lookup
 
-    * Linux/Unix 
-        * Added the time field for both the `Success Login by Host, Users` and `Failed Login by Host, Users` dashboard panels in the `Linux/Unix` dashboard
-        * Removed the `Open Ports` dashboard panel as the `Listening Ports on Host` dashboard panel provides the same information with additional fields.
+    * Replaced `Qualys Host Summary` and `Tenable Vulnerabilities` with a new  `Host Vulnerability Summary` dashboard panel. Similarly replaced `Qualys Vulnerabilities` and `Tenable Summary` with a new `Host Vulnerabilities` dashboard panel in the `Asset Intelligence` dashboard.
```

Comment:
Replaced `Qualys Host Summary` and `Tenable Host Summary` with a new  `Host Vulnerability Summary` dashboard panel. Similarly replaced `Qualys Vulnerabilities` and `Tenable Vulnerabilities` with a new `Host Vulnerabilities` dashboard panel in the `Asset Intelligence` dashboard.